TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bukkit/TimoCloudBukkit.java

```diff
@@ -170,7 +170,10 @@ public void onHandshakeSuccess() {
     }
 
     public void stop() {
-        Bukkit.getScheduler().runTask(this, () -> Bukkit.getServer().shutdown());
+        Bukkit.getScheduler().runTask(this, () -> {
+            Bukkit.getServer().shutdown();
+            System.exit(0);
```

Comment:
Do we always want to do that and not just in case of emergency? We should give plugins time to save their data, etc.